src/main/resources/META-INF/plugin.xml

```diff
@@ -214,6 +218,7 @@
         <javaCodeActionParticipant kind="source"
                                    group="mp"
                                    implementationClass="io.openliberty.tools.intellij.lsp4mp4ij.psi.internal.openapi.java.MicroProfileGenerateOpenAPIOperation"/>
+        <javaWorkspaceSymbolsParticipant implementation="io.openliberty.tools.intellij.lsp4mp4ij.psi.internal.jaxrs.java.JaxRsWorkspaceSymbolParticipant" />
```

Comment:
@aparnamichael @TrevCraw @turkeylurkey I opened https://github.com/OpenLiberty/liberty-tools-intellij/issues/921 to investigate this new function. Was this evaluated? Did we test this? I'm guessing I missed reviewing an intermediate PR that updated LSP4MPiJ but think it would be wise to evaluate any new function coming in before we enable it here.

Reply:
Disabling workspace symbols extension point for now